src/libstd/macros.rs

```diff
@@ -231,10 +231,13 @@ macro_rules! eprintln {
 /// to give up ownership, you can instead borrow with `dbg!(&expr)`
 /// for some expression `expr`.
 ///
+/// The `dbg!` macro works exactly the same in release builds. This is useful when debugging issues
```

Comment:
The textual changes here are good but the indentation is weird as compared with the rest of the text... can you adjust a bit? E.g. split the line after the first `.`

Reply:
Done!

Reply:
I don't think there's any standard. It's mostly to make it work nicely with the rest of the text.

Reply:
There is a standard: https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text

Reply:
(That said, this does not list line lengths, but the standard library wraps to 80 columns as its standard style)

Reply:
@steveklabnik Ahh great! I'll give it a read!